### ISSUE TITLE
for `read*()` functions, check if file exists and output and error if it doesn't 

### ISSUE THREAD

**Issue by taylorreiter** [Arcadia-Science/sourmashconsumr#49]
Otherwise, when a non-existent file is supplied:
```
> Sys.glob("~/github/2022-strains/SRR/SRR*lineage*csv")
character(0)
```
You get this entirely unhelpful error message:
```
Error in tidyr::separate(., .data$lineage, into = c("domain", "phylum", : 
object 'taxonomy_annotate_df' not found
```

**Comment by taylorreiter** [comment]
but...do this cleverly so that if read from URL still works, this check doesn't break that functionality

**Comment by taylorreiter** [comment]
Addressed in #61, including keeping read url functionality
